src/org/sosy_lab/java_smt/test/InterpolatingProverTest.java

```diff
@@ -415,15 +473,21 @@ public <T> void sequentialInterpolationWithOnePartition()
 
     // list of one partition
     List<T> partition = ImmutableList.of(TA, TB);
-    List<BooleanFormula> itps = stack.getSeqInterpolants(ImmutableList.of(partition));
-    assertThat(itps).isEmpty();
+    if (interpolationStrategy == null) {
+      List<BooleanFormula> itps = stack.getSeqInterpolants(ImmutableList.of(partition));
+      assertThat(itps).isEmpty();
+    } else {
+      assertThrows(UnsupportedOperationException.class,
```

Comment:
Should never be reached if `requireIndependentSeqItp()` is correctly implemented.
This goes for all occurrences of this in this class.